src/calculateRentalCost.js

```diff
@@ -4,7 +4,26 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const PRICE = 40;
+  const LONG_TERM_DISCOUNT = 50;
+  const SHORT_TERM_DISCOUNT = 20;
+
+  switch (days) {
+    case 7:
```

Comment:
The constant for the long-term threshold is missing. According to the checklist, you should define a constant for the number of days that qualifies for the long-term discount.